### ISSUE TITLE
Does not handle --pod agrument

### ISSUE THREAD

**Issue by rbadapanda** [alexlafroscia/vim-ember-cli#5]
I might be missing something.

If so can someone share how to do it (and possibly mention this in the readme.md) ?

**Comment by alexlafroscia** [comment]
Yeah, I'm aware of the issue, just haven't had a lot of time to work on the plugin lately.  I'll see what I can do about making that work.

**Comment by anhari** [comment]
Just wanted to add that it does not handle the `--path` argument as well.  (I may be missing something too though)

**Comment by alexlafroscia** [comment]
Nope, not missing anything. It doesn't handle any additional arguments right now 😦

I want to give this some attention but I'm waiting to see what happens with [`ember-language-server`](https://github.com/emberwatch/ember-language-server) first
